specification/mysql/resource-manager/readme.go.md

```diff
@@ -5,25 +5,37 @@ These settings apply only when `--go` is specified on the command line.
 ``` yaml $(go)
 go:
   license-header: MICROSOFT_APACHE_NO_VERSION
-  namespace: mysql
   clear-output-folder: true
 ```
 
 ### Go multi-api
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2020-07-01-privatepreview
   - tag: package-2020-01-01
   - tag: package-2017-12-01-preview
   - tag: package-2017-12-01
 ```
 
+### Tag: package-2020-07-01-privatepreview and go
+
+These settings apply only when `--tag=package-2020-07-01-privatepreview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2020-07-01-privatepreview' && $(go)
+rpname: mysql
+namespace: mysqlflexibleservers
+output-folder: $(go-sdk-folder)/services/preview/$(rpname)/mgmt/2020-07-01-preview/$(namespace)
```

Comment:
I am not quite sure if we could add this new variable here... could we just hard code this rpname in? like this:
```suggestion
namespace: mysql
output-folder: $(go-sdk-folder)/services/preview/mysql/mgmt/2020-07-01-preview/mysqlflexibleservers
```